src/components/paragraphs/sum-calculator/sum-calculator.tsx

```diff
@@ -155,17 +155,8 @@ const SumCalculatorParagraph = ({
               {value: "no1", label: "No, I am a US citizen or permanent US resident"},
               {value: "no2", label: "No, I am an international student with an I-20 sponsored by another institution"},
```

Comment:
Do you want to fix the width while you're in there? 
![image](https://github.com/user-attachments/assets/933e0a57-fd14-4825-bae8-4ce7e58b0cc7)

Reply:
Let's revisit this once #54 has been merged in since that was a FE bug that Yvonne noticed as well.